src/test/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExampleTest.java

```diff
@@ -71,13 +74,24 @@
 
   @ClassRule
   public static final EmbeddedSingleNodeKafkaCluster CLUSTER = new EmbeddedSingleNodeKafkaCluster();
```

Comment:
This final example still depends on the embedded Kafka _broker_, but now uses the mocked SR client.
All the microservice tests will follow this example.